### ISSUE TITLE
Review output format for release

### ISSUE THREAD

**Issue by CharliePoole** [nunit/nunit-summary#9]
Before releasing, but after all the templates are created, let's do a general review comparing the output produced by all eight of them for the same assembly. We should eliminate differences that we didn't intend and make sure all the formats work.

**Comment by rprouse** [comment]
@CharliePoole said,

> You have multiple XML result files and there is no external indication of which ones passed and which one's failed. You use the -brief option with wildcard input and a single file (or the console) for output. The resulting output has just a few lines per test run, which you can easily scan.

That use case makes sense, but to make it usable, we should probably add the test files to the output. As it stands, the summary does not indicate what was tested. We may also want to include the filters.

**Comment by CharliePoole** [comment]
@rprouse Yes... I think that's a difference between the V2 and V3 that we'll have to resolve.

**Comment by CharliePoole** [comment]
I created a temporary page where we can compare the various output formats at http://nunit.org/nunit-summary/

You can independently select what's displayed on the left and on the right side, choosing from the V2 or V3 console output as well as the eight variations of built in reports.

My own comments:
- The V2 reports all display a header line showing the name of the file being tested, while V3 reports don't.
- The V2 and V3 reports have arbitrary differences - that is, there are some differences that result from added features in V3 but others are just different.

Please make your own comparisons and let's discuss.
